changelogs/fragments/62587-module_utils-network-cloudengine.yml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+  - "Cloudengine module_utils - the 'set-id'(RPC-REPLY XML attribute) may change over the time althougth 'set-id' is the indentity of next RPC packet."
+  - "Cloudengine netconf plugin - add a dispatch RPC function to send and receive original RPC reply."
```

Comment:
```suggestion
```
This is an implementation detail which isn't interesting for users.

Reply:
em, I have updated this.

Reply:
Yeah, the PR has been validated many times on real devices befor it was ready for review.
Every time I commit some codes after I tested them with real hardware.
But there  are some private policyices about codes and devices information that can not send them to plulic network and comment it here.
No test, no commiting.and also our leader @hwDCN watch the PR.